queries/julia/injections.scm

```diff
@@ -1,4 +1,5 @@
-((triple_string) @markdown
-  (#offset! @markdown 0 3 0 -3))
+; TODO: re-add when markdown is added.
+; ((triple_string) @markdown
```

Comment:
But why remove it then? Who want markdown can actually use it.

Reply:
Checks will fail as markdown isn't declared in `parsers.lua`.